src/NuGetForUnity/Editor/Ui/NugetPreferences.cs

```diff
@@ -35,14 +35,22 @@ public class NugetPreferences : SettingsProvider
         /// </summary>
         public const string NuGetForUnityVersion = "4.0.2";
 
+        private const float LablePading = 5;
```

Comment:
Typo, should be LabelPadding I guess.

Reply:
This typo often happens on my side (not sure why)